certbot/CHANGELOG.md

```diff
@@ -14,7 +14,8 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 
 ### Fixed
 
-*
+* Certbot won't parse the configuration files located at the default configuration file locations
+  any longer if the `-c` or `--config` command line option has been specified.
```

Comment:
I am not sure, but I am worried that this is a breaking change.

If a user was (perhaps without realizing) relying on the merge behavior previously, it might not be safe to suddenly change the behavior here.

I will find out for you.

Reply:
btw I have not forgotten about this PR, need to check some things!

Reply:
Marking this as "resolved" as it has been decided to defer this PR to 2.0, which, if I'm interpreting this deference correctly, allows for breaking changes.